src/lib/octokit.js

```diff
@@ -8,11 +8,23 @@ const RETRY_LIMIT = 5
 
 let client
 
+const labels = new Set()
+
+const clearPendingMessages = (status = 'done') => {
+	labels.forEach(label => {
+		logger.log(label, { status: status })
+	})
+	labels.clear()
+}
+
 const retryWrapper = (retryAfter, options, message) => {
 	if (options.request.retryCount === RETRY_LIMIT) {
+		clearPendingMessages('fail')
```

Comment:
we need to call clearPendingMessages here as well, because after we hit the RETRY_LIMIT octokit throws an error to stop the retrying, which means that the clearPendingMessages in prs.js won't get hit. By calling this function here we transition the pending messages to fail messages
![Screenshot 2021-12-09 at 18 11 45](https://user-images.githubusercontent.com/20088576/145453744-c242b3b7-8ed9-4100-bdad-c414e8f77bc5.png)